src/js/components/awardv2/AwardV2.jsx

```diff
@@ -80,6 +90,10 @@ export default class Award extends React.Component {
 
         const sectionTop = sectionDom.offsetTop - 145;
         scrollToY(sectionTop, 700);
+        if (updateState) {
```

Comment:
Lets keep this function only dedicated to `jumpingToSection` and have a new function you can compose together with this one that is dedicated to updating state.

Just by looking at this function, it's hard to read what is happening here.